consumers/evm.md

```diff
@@ -4,13 +4,34 @@ description: Consume Pyth Network prices in applications on EVM-based chains
 
 # Pyth on EVM
 
+## How Pyth Works on EVM Chains
+
+Pyth prices are published on Solana and relayed to EVM chains using the [Wormhole Network](https://wormholenetwork.com/) as a cross-chain message passing bridge. The Wormhole Network observes when Pyth prices on Solana have changed and publishes an off-chain signed message attesting to this fact. This process is explained in more detail [here](https://docs.wormholenetwork.com/wormhole/).
+
+This signed price update message can then be submitted to the Pyth contract on the EVM networks, verifying the Wormhole message and updating the Pyth contract with the new price.
+
+Protocols integrating with Pyth can update the on-chain Pyth prices in two different ways. They can update the prices on demand or regularly,
+as described in more detail below.
+
+
+### On-demand price updates
+
+Price updates are not automatically submitted on the EVM networks: instead, when a consumer needs to use the value of a price, they should first submit the latest Wormhole update for that price to the Pyth contract on their target EVM network. Updating prices on-demand will make the most recent price update available on-chain for EVM contracts to use. In addition, on-demand price updates minimize latency and are more gas efficient, as prices are only updated on-chain when needed.
```

Comment:
this paragraph seems out of place. I think you need to say that protocols should craft a single transaction that (1) updates the on-chain pyth price, and (2) does whatever the protocol action does. If you want to explain why we don't continuously update the on-chain price, I would do that in the section above explaining how pyth works.